server/src/main.rs

```diff
@@ -75,11 +73,13 @@ fn main() -> Result<(), String> {
         fs::create_dir_all(data_folder_path.as_path())
             .map_err(|error| format!("Unable to create {}: {}", &data_folder, error))?;
 
+        // Create a tokio runtime for executing asynchronous tasks.
```

Comment:
Maybe replace "tokio" with "Tokio" for consistency and add an extra sentence to the comment stating why the runtime is not part of the context so we do not get the bright idea in the future to try and make it part of the context again.

Reply:
Fixed.